src/main/app/package.json

```diff
@@ -23,8 +23,8 @@
     "@angular/core": "^17.3.12",
     "@angular/forms": "^17.3.12",
     "@angular/google-maps": "^17.2.0",
-    "@angular/material": "^17.2.0",
-    "@angular/material-moment-adapter": "^17.2.0",
+    "@angular/material": "^17.3.6",
+    "@angular/material-moment-adapter": "^17.3.10",
```

Comment:
Should this upgrade not be part of a different PR? Why is this not done by Renovate or Dependabot?

Reply:
it seems like Renovate and Dependabot are not taking the angular app into account 😟